### ISSUE TITLE
What is the rescaled_phi in classifier_free_guidance.py 

### ISSUE THREAD

**Issue by wbensvage** [lucidrains/denoising-diffusion-pytorch#253]
In [classifier_free_guidance.py](https://github.com/lucidrains/denoising-diffusion-pytorch/blob/main/denoising_diffusion_pytorch/classifier_free_guidance.py) , after getting 

> scaled_logits = null_logits + (logits - null_logits) * cond_scale
in function "forward_with_cond_scale"

there are still some lines doing:
`        if rescaled_phi == 0.:
            return scaled_logits

        std_fn = partial(torch.std, dim = tuple(range(1, scaled_logits.ndim)), keepdim = True)
        rescaled_logits = scaled_logits * (std_fn(logits) / std_fn(scaled_logits))

        return rescaled_logits * rescaled_phi + scaled_logits * (1. - rescaled_phi)`

May I ask what is the rescaled_phi? And is there any paper/article/blog/etc. mentioned this?

**Comment by lucidrains** [comment]
it is from https://arxiv.org/abs/2305.08891, a bytedance AI labs paper

**Comment by wbensvage** [comment]
Thx @lucidrains!

**Comment by lucidrains** [comment]
@wbensvage no problem, happy AI training
